python/revng/internal/cli/_commands/mass_testing/configure.py

```diff
@@ -24,6 +24,7 @@ def register_arguments(self, parser: argparse.ArgumentParser):
         parser.description = "Collect binary files and run test-configure"
         parser.add_argument("--meta", help="Metadata definition")
         parser.add_argument("--seed", help="Seed to use in the RNG for shuffling")
+        parser.add_argument("--sample", help="Sample the inputs to the specified amount")
```

Comment:
Is this for debugging purposes?

Reply:
Yes, I needed to check if `mass-testing` was working but didn't want to wait for 2 days. I've left it in the PR since it might be useful in the future.